sdk/eventhub/tests.yml

```diff
@@ -7,4 +7,4 @@ extends:
     ServiceDirectory: eventhub
     SDKType: client
     TimeoutInMinutes: 190
```

Comment:
We probably need to bump this to 240;  China ARM operations appear to be very, very slow and the runs there have been timing out.